src/pages/List.jsx

```diff
@@ -1,3 +1,75 @@
+import { useRef } from "react";
+import SupportCard from "../components/Card/SupportCard";
+import styles from "./List.module.scss";
+import ArrowButton from "../components/Button/ArrowButton/ArrowButton";
+
 export default function List() {
-  return <></>;
+  const slider = useRef();
+  let isDown = false;
+  let startX;
+  let scrollLeft;
+
+  const handleMouseDown = (e) => {
+    isDown = true;
+    startX = e.pageX - slider.current.offsetLeft;
+    scrollLeft = slider.current.scrollLeft;
+  };
+
+  const handleMouseLeave = (e) => {
+    isDown = false;
+  };
+
+  const handleMouseUp = (e) => {
+    isDown = false;
+  };
+
+  const handleMouseMove = (e) => {
+    if (!isDown) return;
+    e.preventDefault();
+    const x = e.pageX - slider.current.offsetLeft;
+    const walk = x - startX;
+
+    slider.current.scrollLeft = scrollLeft - walk;
+  };
```

Comment:
상태를 let으로 설정하신 이유가 따로 있으실까요?
문제가 없어보이긴 하지만 리액트에서 상태 관리 데이터는 useState로 관리하는 것이 더 일반적입니다

```suggestion
  const [isDown, setIsDown] = useState(false);
  const [startX, setStartX] = useState(null);
  const [scrollLeft, setScrollLeft] = useState(null);
  const slider = useRef();

  const handleMouseDown = (e) => {
    setIsDown(true);
    setStartX(e.pageX - slider.current.offsetLeft);
    setScrollLeft(slider.current.scrollLeft);
  };

  const handleMouseLeave = () => {
    setIsDown(false);
  };

  const handleMouseUp = () => {
    setIsDown(false);
  };

  const handleMouseMove = (e) => {
    if (!isDown) return;
    e.preventDefault();
    const x = e.pageX - slider.current.offsetLeft;
    const walk = x - startX;

    slider.current.scrollLeft = scrollLeft - walk;
  };
```

따로 이유가 없으시다면 위와 같이 변경하는 것을 제안드려요!

Reply:
자세한 설명 감사합니다 👍